apps/payments/next/app/[offeringId]/checkout/page.tsx

```diff
@@ -0,0 +1,47 @@
+import { getContentfulContent, getCartData } from '../../lib/apiClient';
+import { PurchaseDetails, TermsAndPrivacy } from '@fxa/payments/ui/server';
+
+interface CheckoutParams {
+  offeringId: string;
+}
+
+export default async function Index({ params }: { params: CheckoutParams }) {
+  // TODO - Fetch Cart ID from cookie
+  // https://nextjs.org/docs/app/api-reference/functions/cookies
+  const cartId = 'cart-uuid';
+  // TODO - Fetch locale from params
+  // Possible solution could be as link below
+  // https://nextjs.org/docs/app/building-your-application/routing/internationalization
+  const locale = 'en-US';
+
+  const contentfulData = getContentfulContent(params.offeringId, locale);
+  const cartData = getCartData(cartId);
+  const [contentful, cart] = await Promise.all([contentfulData, cartData]);
+
+  return (
+    <>
+      <h1 className="page-title-container">Under Construction</h1>
+      <section className="payment-panel" aria-label="Purchase details">
+        <PurchaseDetails
+          interval={cart.interval}
+          invoice={cart.nextInvoice}
+          purchaseDetails={contentful.purchaseDetails}
+        />
+      </section>
+      <section
```

Comment:
Should this `section` be `main`?
cc: @sardesam

Reply:
See [this comment](https://github.com/mozilla/fxa/pull/15710#discussion_r1316344411) above.